### ISSUE TITLE
Investigate Bitbucket

### ISSUE THREAD

**Issue by Jameskmonger** [himynameisdave/git-labelmaker#55]
I think it would be good to investigate [Bitbucket](https://bitbucket.org/) as a potential integration.

There doesn't appear to be a "Label" system on Bitbucket that goes by the same name, however there is [apparently a "components" system which works in the same way](https://bitbucket.org/site/master/issues/5356/labels-for-issues-bb-6550).

Bitbucket is used by a lot of people/companies with [JIRA](https://www.atlassian.com/software/jira) and so it would be great if we could integrate.

Like #48, this isn't a priority.

**Comment by himynameisdave** [comment]
Agreed, this tool is called **git-**labelmaker so it should technically support all types of Git managers/softwares like GitLab and BitBucket
